package.json

```diff
@@ -21,6 +22,7 @@
     "react": "^16.13.1",
     "react-app-polyfill": "^1.0.6",
     "react-aria-menubutton": "^7.0.0",
+    "react-collapsed": "^3.0.0",
```

Comment:
How do you know about all of these libraries?!?!  Are you just googling around until you find something or have you worked with them in the past?

Reply:
i am absolutely just frantically searching npm

Reply:
![frantic search](https://media.giphy.com/media/clupdT5vHL9GifMlnC/giphy.gif)